### DISCUSSION TITLE
Student Developer Pack access issue.

### DISCUSSION THREAD

**Discussion by Shahariar751** [community/community#77108] [GitHub Education]
### Select Topic Area

Bug

### Body

So I've been been verified as a student but can't access to the student developer pack. When I click on my student benefits they say that I can now view my benefits but in reality I'm having trouble accessing my developer pack. When I re-inter my information they say I don't need to do that as I'm already verified and have been approved for benefits. Can anyone please look into this issue?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Shahariar751 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by praticodes** [comment]
The exact same thing has happened to me. Let me know how you can fix it

**Reply by praticodes** [reply]
@queenofcorgis I am having the exact same problem
